src/main/java/SYSC6/Project/Main_Controller.java

```diff
@@ -159,8 +165,6 @@ public String getUsers(){
         return logout();
     }
 
-    //---
-
```

Comment:
Just FYI, this may cause a merge conflict with aiman's PR but shouldn't be a big deal since you're just removing an unnecessary comment.